example/example.cfg.example

```diff
@@ -27,6 +28,12 @@ MULTIAUTH_AUTH_PROVIDERS = {
     'local': {
         'type': 'example_local',
         'title': 'Local Accounts'
+    },
+    'my_shibboleth': {
+        'type': 'shibboleth',
+        'title': 'Shibboleth',
+        'callback_uri': '/shibboleth/sso',
```

Comment:
I wouldn't put this in the example config. In most cases the generated name is fine and documenting what the URL will be and that it needs to be configured in the shibboleth section of the Apache config should be enough.

Another option would be removing the default value for it and raising an exception if it's not defined... Having to explicitly specify the URL in any case wouldn't be so bad...